qiime/core/type/signature.py

```diff
@@ -133,12 +146,12 @@ def _parse_signature(self, callable, inputs, parameters, outputs):
                                     " parameters in callable signature.")
                 annotated_inputs[name] = ParameterSpec(
                     qiime_type=inputs.pop(name), view_type=view_type,
-                    default=default)
+                    default=default, help_text=help_text.get(name))
```

Comment:
Yup! That's why we use a sentinel value at all, because for this particular API `None` *is* a real value, so we have to once more disambiguate between "nothing" and "no really, nothing"